### PR TITLE
Update zh-CN.txt

### PR DESCRIPTION
**Pull request by tkliuxing**
Have done some translation.

### CODE REVIEW

File: data/language/zh-CN.txt [hunk 1]
```diff
@@ -4746,19 +4746,305 @@ STR_SCNR    :Blackpool Pleasure Beach
...
+STR_SCNR    :Roaring Twenties - Skyscrapers
+STR_PARK    :Metropolis
+STR_DTLS    :你拥有一个空的地段，靠近城镇的低矮的部分。把最贵的城市属性，建立一个主题公园的启发，摩天大楼高耸的建筑装饰艺术的年代。
+
+<Rock 'n' Roll - Flower Power>
+STR_SCNR    :Rock 'n' Roll - Flower Power
+STR_PARK    :Woodstock
+STR_DTLS    :一个大型的年度音乐节在你的土地上。打造一个主题公园，保持自由奔放的观众的娱乐。
 
-#WW
```
Comment (Gymnasiast) [inline data/language/zh-CN.txt]: Why did you remove this?

Comment (tkliuxing) [inline data/language/zh-CN.txt]: It's my fault. I accidentally deleted #WW when I added some .DAT strings.

File: data/language/zh-CN.txt [hunk 2]
```diff
@@ -1178,8 +1178,8 @@ STR_1167    :Can't raise water level here...
 STR_1168    :选项
 STR_1169    :(无)
 STR_1170    :{STRING}
-STR_1171    :{RED}已关闭 - - 
-STR_1172    :{YELLOW}{STRINGID} - - 
+STR_1171    :{RED}已关闭 - -
```
Comment (Gymnasiast) [inline data/language/zh-CN.txt]: Restore these. The trailing whitespace belongs here.

File: data/language/zh-CN.txt [hunk 3]
```diff
@@ -1350,10 +1350,10 @@ STR_1339    :{BLACK}No test results yet...
 STR_1340    :{WINDOW_COLOUR_2}Max. speed: {BLACK}{VELOCITY}
 STR_1341    :{WINDOW_COLOUR_2}Ride time: {BLACK}{STRINGID}{STRINGID}{STRINGID}{STRINGID}
 STR_1342    :{DURATION}
-STR_1343    :{DURATION} /
```
Comment (Gymnasiast) [inline data/language/zh-CN.txt]: Same for this one and 1346.

File: data/language/zh-CN.txt [hunk 4]
```diff
@@ -1738,8 +1738,8 @@ STR_1726    :Land not for sale!
 STR_1727    :Construction rights not for sale!
 STR_1728    :Can't buy construction rights here...
 STR_1729    :Land not owned by park!
-STR_1730    :{RED}Closed - - 
-STR_1731    :{WHITE}{STRINGID} - - 
+STR_1730    :{RED}Closed - -
```
Comment (Gymnasiast) [inline data/language/zh-CN.txt]: Restore these two.

File: data/language/zh-CN.txt [hunk 5]
```diff
@@ -1781,12 +1781,12 @@ STR_1769    :{WINDOW_COLOUR_2}Number of swings:
...
 STR_1772    :{COMMA16}
-STR_1773    :每个游乐设施只可建造一个照相区域
-STR_1774    :每个游乐设施只可建造一个缆索坡道
+STR_1773    :每个游乐设施只能建造一个照相区域
+STR_1774    :每个游乐设施只能建造一个牵引轨道
 STR_1775    :关闭
 STR_1776    :开启
 STR_1777    :{WINDOW_COLOUR_2}设施音乐
-STR_1778    :{STRINGID} - - 
+STR_1778    :{STRINGID} - -
```
Comment (Gymnasiast) [inline data/language/zh-CN.txt]: And this one.

File: data/language/zh-CN.txt [hunk 6]
```diff
@@ -2795,8 +2795,8 @@ STR_2779    :查看窗口 #{COMMA16}
 STR_2780    :额外的查看窗口
 # End of new strings
 STR_2781    :{STRINGID}:{MOVE_X}{195}{STRINGID}
-STR_2782    :SHIFT + 
-STR_2783    :CTRL + 
+STR_2782    :SHIFT +
```
Comment (Gymnasiast) [inline data/language/zh-CN.txt]: And these two.

File: data/language/zh-CN.txt [hunk 7]
```diff
@@ -3176,13 +3176,13 @@ STR_3159    :list
 STR_3160    :{SMALLFONT}{BLACK}选择每次乘坐循环的次数
 STR_3161    :<not used anymore>
 STR_3162    :无法分配足够的内存
-STR_3163    :安装新数据档中: 
+STR_3163    :安装新数据档中:
 STR_3164    :{BLACK}已选择{COMMA16}项 (最多{COMMA16}项)
 STR_3165    :
-STR_3166    :{BLACK}(ID: 
+STR_3166    :{BLACK}(ID:
```
Comment (Gymnasiast) [inline data/language/zh-CN.txt]: And this one.

File: data/language/zh-CN.txt [hunk 8]
```diff
@@ -4196,7 +4196,7 @@ STR_5880    :只显示选中的物体
 STR_5881    :只显示未被选中的物体
 STR_5882    :自定义货币
 STR_5883    :自定义货币选项
-STR_5884    :{WINDOW_COLOUR_2}汇率: 
+STR_5884    :{WINDOW_COLOUR_2}汇率:
```
Comment (Gymnasiast) [inline data/language/zh-CN.txt]: And this one.

**Comment by Gymnasiast** [comment]
Please turn off whitespace correction, apart from one they were all needed. Please restore them.

**Comment by tkliuxing** [comment]
Restore is complete.

**Comment by Gymnasiast** [comment]
Thank you!
